sound/soc/intel/common/soc-acpi-intel-tgl-match.c

```diff
@@ -56,6 +56,38 @@ static const struct snd_soc_acpi_adr_device rt1308_1_adr[] = {
 	}
 };
 
+static const struct snd_soc_acpi_adr_device rt1308_1_single_adr[] = {
+	{
+		.adr = 0x000120025D130800,
+		.num_endpoints = 1,
+		.endpoints = &single_endpoint,
+	},
+};
+
+static const struct snd_soc_acpi_adr_device rt1308_1_group1_adr[] = {
+	{
+		.adr = 0x000110025D130800,
```

Comment:
I think the version id is 2? ie ``.adr = 0x000120025D130800`` like the adr in rt1308_1_single_adr[]. Actually, I see a lot of 10025D in match tables when I double check the sdw adrs.

Reply:
On TGL RVP, version = 2
````
static const struct snd_soc_acpi_adr_device rt1308_1_adr[] = {
	{
		.adr = 0x000120025D130800,
		.num_endpoints = 1,
		.endpoints = &spk_l_endpoint,
	},
	{
		.adr = 0x000122025D130800,
		.num_endpoints = 1,
		.endpoints = &spk_r_endpoint,
	}
};
````

Reply:
@shumingfan can you help here? Do we really have two versions of RT1308?

Reply:
> @shumingfan can you help here? Do we really have two versions of RT1308?

@shumingfan Not just rt1308 but also rt711 and rt715, please.

Reply:
> @shumingfan can you help here? Do we really have two versions of RT1308?

@RanderWang I think the bit-fields with the different values should be UniqueId, right?
The UniqueId of RT1308 could be controlled by ASEL1/ASEL0 pins.
ASEL1   ASEL0  UniqueId
   0           0             0
   0           1             1    
   1           0             2
   1           1             3
You could check the connection with ASEL0/1 pins.

Reply:
> @shumingfan Not just rt1308 but also rt711 and rt715, please.

Regarding RT711, I checked the spec. the uniqueId also is controlled by Pad MHDA_SYNC, MHDA_SDATA-OUT.
RT715 only has two options (0 or 1). It controls by pin GPIO_5(DMIC_CLK2).

Reply:
> I think the version id should be 2? ie `.adr = 0x000120025D130800` like the adr in rt1308_1_single_adr[]. Actually, I see a lot of 10025D in match tables when I double check the sdw adrs.

About SdW version, RT1308 should be 2 (SdW v1.1).
The RT711/715 has two SdW versions including 2 (SdW v1.1) and 3 (SdW v1.2).